docs/getting-started.md

```diff
@@ -0,0 +1,38 @@
+# Getting started
+
+`fcct` is a tool that will consume a Fedora CoreOS Config and produce a JSON file that can be given to a Fedora CoreOS machine when it first boots to configure the machine. Using this config, a machine can be told to create users, create filesystems, set up the network, install systemd units, and more.
```

Comment:
Maybe: "`fcct`, the Fedora CoreOS Config Transpiler, is a tool that consumes a Fedora CoreOS Config and produces an Ignition config, which is a JSON document that can be given to a Fedora CoreOS machine to configure the machine when it first boots."

A machine being `told to` do things is not declarative, but never mind.